lib/web/appserver.rb

```diff
@@ -551,6 +551,17 @@ def start_device_ejectable_event
     end
   end
 
+  post "/api/update_by_tag" do
+    tags = params["tags"] or pass
+    tag_params = tags.map do |tag|
+      "tag:#{tag}"
+    end
+    Narou::Worker.push do
+      CommandLine.run!(["update", tag_params])
+      @@push_server.send_all(:"table.reload")
```

Comment:
__[reek]__
* [ClassVariable] declares the class variable @@push_server [:ref](https://github.com/troessner/reek/blob/master/docs/Class-Variable.md)